oaipmhTemplates/dha_RepoObject.xml

```diff
@@ -0,0 +1,154 @@
+        </dha:hasPreservationDescription>
+
+        <dha:hasDisseminationDescription rdf:parseType="Resource"> <!-- Q: Would it be possible to list all Dissemination Services and include each of them as separate objects, by repeating the template  in the element <dha:hasDisseminationService>? -->
+            <dha:hasDisseminationService rdf:parseType="Resource">
+                <premis:hasIdentifier/> <!-- Q: URI to Dissemination Service for this object. For images, IIIFURL could be used here? -->
+                <dha:hasAnchorLabel/> <!-- Q: This is something like a literal label: how can I insert the title of the dissemination service? -->
+                <dha:conformsTo count="*" val=""/> <!-- Q: In the examples there is something like &dha;/service/api/get - maybe we could put something similar from ARCHE? -->
+                <dha:hasMimeType count="*" val="/acdh:hasFormat"/>
+            </dha:hasDisseminationService>
+        </dha:hasDisseminationDescription>
+
+        <dha:hasAnnotationDescription rdf:parseType="Resource">
+
+                <dha:Concept rdf:parseType="Resource">
+                    <dha:hasCatchPhrase count="*" val="/acdh:hasSubject"></dha:hasCatchPhrase>
+                    <!-- The following part doesn't apply, since subjects are literals in ARCHE:
+
+                      <dha:hasIdentifier rdf:parseType="Resource">
+                          <premis:hasIdentifier></premis:hasIdentifier>
+                          <dha:conformsTo></dha:conformsTo>
+                          <dha:hasParsableMetadata></dha:hasParsableMetadata>
+                      </dha:hasIdentifier>
+
+                    -->
+                </dha:Concept>
+
+                <dha:NamedEntity rdf:parseType="Resource">
+
+                    <dha:Artefact rdf:parseType="Resource">
+                        <dha:hasCatchPhrase></dha:hasCatchPhrase>
+                        <dha:hasIdentifier rdf:parseType="Resource">
+                            <premis:hasIdentifier></premis:hasIdentifier>
+                            <dha:conformsTo></dha:conformsTo>
+                            <dha:hasParsableMetadata></dha:hasParsableMetadata>
+                        </dha:hasIdentifier>
+                    </dha:Artefact>
+
+                    <dha:Organization rdf:parseType="Resource"> <!-- Q: How do distinguish between hasActor that points to an Organization rather than a Person? Use If conditions? -->
+                        <dha:hasCatchPhrase count="*" lang="true" val="@acdh:hasActor/hasTitle"></dha:hasCatchPhrase>
+                        <dha:hasIdentifier rdf:parseType="Resource">
+                            <premis:hasIdentifier count="1" val="@acdh:hasActor/hasIdentifier"></premis:hasIdentifier>
+                            <dha:conformsTo>https://id.acdh.oeaw.ac.at/</dha:conformsTo>
+                            <dha:hasParsableMetadata count="*" val="METAURL"></dha:hasParsableMetadata>
```

Comment:
Here you probably want the metadata URL of the target organization but the `METAURL` would give you metadata URL of the processed repository resource. Use subtemplates to achieve what you want. They allow you to "move" to another resource.

Reply:
Ah sure, right! 🤦‍♂️